### PR TITLE
1.3.x fix https://issues.redhat.com/browse/MODCLUSTER-712 

### PR DESCRIPTION
**Pull request by jfclere**
remove httpd-2.2.x code.

### CODE REVIEW

**Comment by jfclere** [comment]
Retest this please.

**Comment by rhusar** [comment]
Is any of this relevant to forward port to master? I assume this was already done in master.

**Comment by rhusar** [comment]
Also, sounds like these two commits can be squashed.

**Comment by jfclere** [comment]
@rhusar yes it is already in master.

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 1]
```diff
@@ -58,17 +58,18 @@
...
+#else
+#error "httpd 2.2.x NOT SUPPORTED!"
+#endif
+
 struct proxy_cluster_helper {
     int count_active; /* currently active request using the worker */
-#if AP_MODULE_MAGIC_AT_LEAST(20051115,4)
-#else
     apr_interval_time_t ping_timeout;
     char ping_timeout_set;
```
[the remark this answers is not in the document]
Comment (jfclere) [reply native/mod_proxy_cluster/mod_proxy_cluster.c]: fixed thanks.

**Pull request merged**